Remove nc_set_log_level() because it is not (no longer?) available in libnetcdf
--- ni/src/ncl/NclNetCDF4.c.orig	2019-02-28 00:44:39
+++ ni/src/ncl/NclNetCDF4.c	2025-02-19 14:07:33
@@ -409,7 +409,6 @@
     }
 
     /*nc_set_log_level(3);*/
-    nc_set_log_level(3);
 
     ChunkSizeHint = 2 * blksize;
 
--- ni/src/ncl/NclNetCdf.c.orig	2019-02-28 00:44:39
+++ ni/src/ncl/NclNetCdf.c	2025-02-19 14:07:38
@@ -614,7 +614,6 @@
 	}
 
         /*nc_set_log_level(3);*/
-        nc_set_log_level(3);
 
 	ChunkSizeHint = 64 * blksize;
 	therec = (NetCdfFileRecord*)NclCalloc(1, sizeof(NetCdfFileRecord));
--- ni/src/ncl/NclNewHDF5.c.orig	2019-02-28 00:44:39
+++ ni/src/ncl/NclNewHDF5.c	2025-02-19 14:08:16
@@ -3834,7 +3834,6 @@
     }
 
     /*nc_set_log_level(3);*/
-    nc_set_log_level(3);
 
     grpnode = (NclFileGrpNode *)NclCalloc(1, sizeof(NclFileGrpNode));
     assert(grpnode);
--- ni/src/ncl/NclNewHDFEOS5.c.orig	2019-02-28 00:44:39
+++ ni/src/ncl/NclNewHDFEOS5.c	2025-02-19 14:08:21
@@ -530,7 +530,6 @@
     }
 
     /*nc_set_log_level(3);*/
-    nc_set_log_level(3);
 
     grpnode = (NclFileGrpNode *)NclCalloc(1, sizeof(NclFileGrpNode));
     assert(grpnode);
